Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "quickcheck"
-version = "0.5.0"  #:version
+version = "0.5.1"  #:version
```

Comment:
Please don't version bump. I do this as part of releases.

Reply:
Reverted. I only bumped because anyone on 0.5.0 will still have a broken build